docs/client/full-api/tableOfContents.js

```diff
@@ -311,12 +312,14 @@ var toc = [
     "appcache",
     "accounts-ui",
     "audit-argument-checks",
+    {name: "blaze", link: "https://atmospherejs.com/meteor/blaze"},
```

Comment:
Are there any advantages to keeping Blaze and Spacebars?

2015-03-17 13:45 GMT-07:00 Sashko Stubailo <notifications@github.com>:

> In docs/client/full-api/tableOfContents.js
> <https://github.com/meteor/meteor/pull/3919#discussion_r26614185>:
>
> > @@ -311,12 +312,14 @@ var toc = [
> >      "appcache",
> >      "accounts-ui",
> >      "audit-argument-checks",
> > +    {name: "blaze", link: "https://atmospherejs.com/meteor/blaze"},
>
> Let's get rid of Blaze and Spacebars in this list. This part of the docs
> so far has been a dumping ground for things we couldn't put elsewhere, and
> we want to reduce its size rather than increase it if possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/meteor/pull/3919/files#r26614185>.
>